extensions/binding/org.eclipse.smarthome.binding.tradfri/src/main/java/org/eclipse/smarthome/binding/tradfri/handler/TradfriLightHandler.java

```diff
@@ -114,7 +114,7 @@ private void setColorTemperature(PercentType percent) {
     private void setColor(HSBType hsb) {
         TradfriLightData data = new TradfriLightData();
         data.setColor(hsb).setTransitionTime(DEFAULT_DIMMER_TRANSITION_TIME);
-        set(data.getJsonString(), 1000);
```

Comment:
The delay was added by intention, see this change set and especially the removed comment about setting color and brightness: https://github.com/eclipse/smarthome/pull/4299/files

Reply:
My question is: When setting the brightness later which will interrupt the color fading, why not set the brightness first and then set the color and let the bulb fade to the new color in the target brightness.

Reply:
Yes, I can remember that discussion. Beside the delay of 1000ms for color commands a default delay of 500ms for every command has been introduced. This has been changed later to 600ms (see #4388). My question is? Why do we need 1000ms for a color command. Is the default delay not sufficient? This PR reduces the delay for color commands from 1000ms to 600ms.

> why not set the brightness first and then set the color

From my point of view that does not make a difference. We have to wait for both resulting states at least 600ms. The real feel for the user is worse in both cases.

Reply:
In this case we can omit the whole `#set(String, int)` infrastructure as it is only used by the setColor call and fall back to the 600ms default delay. wdyt?

Reply:
imho you can rework the whole thing. Otherwise the code would be unused. So dump just dump it.

Reply:
Done.